### PR TITLE
add hyperlink

### PR DESCRIPTION
**Pull request by Beginner-Go**

### CODE REVIEW

**Review by pan93412 — APPROVED** [review]

File: luasrc/model/cbi/unblockneteasemusic/unblockneteasemusic.lua [hunk 1]
```diff
@@ -1,5 +1,6 @@
-mp = Map("unblockneteasemusic", translate("解除网易云音乐播放限制"))
-mp.description = translate("原理：采用 [Bilibili/JOOX/酷狗/酷我/咪咕/pyncmd/QQ/Youtube] 等音源，替换网易云音乐 无版权/收费 歌曲链接<br/>具体使用方法参见：https://github.com/UnblockNeteaseMusic/luci-app-unblockneteasemusic")
+mp = Map("unblockneteasemusic")
+mp.title = translate("解除网易云音乐播放限制")
```
Comment (1715173329) [inline luasrc/model/cbi/unblockneteasemusic/unblockneteasemusic.lua:2]:
```suggestion
mp = Map("unblockneteasemusic", translate("解除网易云音乐播放限制"))
```
Keep it as-is.

File: luasrc/model/cbi/unblockneteasemusic/unblockneteasemusic.lua [hunk 2]
```diff
@@ -1,5 +1,6 @@
...
+mp.description = translate("原理：采用 [Bilibili/JOOX/酷狗/酷我/咪咕/pyncmd/QQ/Youtube] 等音源，替换网易云音乐 无版权/收费 歌曲链接<br/>具体使用方法参见：<a href=\"https://github.com/UnblockNeteaseMusic/luci-app-unblockneteasemusic\" target=\"_blank\">GitHub 项目地址</a>")
```
Comment (1715173329) [inline luasrc/model/cbi/unblockneteasemusic/unblockneteasemusic.lua:3]:
```suggestion
mp.description = translate("原理：采用 [Bilibili/JOOX/酷狗/酷我/咪咕/pyncmd/QQ/Youtube] 等音源，替换网易云音乐 无版权/收费 歌曲链接<br/>具体使用方法参见：<a href=\"https://github.com/UnblockNeteaseMusic/luci-app-unblockneteasemusic\" target=\"_blank\">GitHub @UnblockNeteaseMusic/luci-app-unblockneteasemusic</a>")
```

**Review by 1715173329 — CHANGES_REQUESTED** [review]
btw, if you would like to use hyperlink, please apply it to other urls as well.

**Comment by Beginner-Go** [comment]
Has been changed to what you expected

**Review by 1715173329 — APPROVED** [review]

**Pull request merged**